workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -291,6 +293,22 @@ res.list[['lfc2']] <- list(
 )
 ```
 
+```{r assemble_variables, dependson=c('results', 'results2'), cache=TRUE}
+# NOTE: Add a new dependson for every contrast chunk to prevent this from
+# malfunctioning.
+# res.list is a named list. Each item should be a list with names c('res',
+# 'dds', 'label'). "res" is a DESeqResults object, "dds" is the corresponding
+# DESeq object the results were extracted from, and "label" is a nicer label to
+# use for headers and other text.
```

Comment:
this should go in those docs too

Reply:
@njohnso6 I think you can use `dependson=knitr::all_labels()[grepl('results', knitr::all_labels())]` to handle this automatically.